src/app/AppHeader.vue

```diff
@@ -119,7 +118,6 @@ const [
 const store = useStore()
 const env = useEnv()
 
-const shouldShowAppError = computed(() => store.getters.shouldShowAppError)
```

Comment:
As our version is now burnt into the kuma binary/GUI, whether we get an error from the API has no bearing on whether we show you your current version or not.

In fact I just realized a UX advantage that if there is an error, you probably want to know what version of kuma you are running.